src/components/Theme/constants.css

```diff
@@ -9,3 +9,7 @@
   bottom: 0;
   right: 0;
 }
+
+.width-parent-relative(@base-width,@parent-width) {
```

Comment:
possible to put this in a `utils.css`? Theme folder is fine for now I think.

Reply:
Yes, I've created a [ticket](https://jira.onfido.co.uk/browse/CX-1578) for that.